sympy/core/tests/test_args.py

```diff
@@ -1710,6 +1710,14 @@ def test_sympy__functions__special__hyper__HyperRep_sinasin():
     assert _test_args(HyperRep_sinasin(x, y))
 
 
+@XFAIL
```

Comment:
This is not good. We shouldn't be creating new Basic subclasses that don't satisfy the args invariants.